src/main/java/com/jalasoft/sdfc/pages/AppLaunchPage.java

```diff
@@ -0,0 +1,23 @@
+package com.jalasoft.sdfc.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AppLaunchPage extends AbstractPage {
+    private static final String PRIVACY_CSS = "//a[@title='%s']";
```

Comment:
I think it should be called ITEM_LINK or something like that

Reply:
Could be "SETUP_ITEM_LINK_CSS" instead PRIVACY_CSS

Reply:
please review the last commit